python/lsst/meas/base/algorithm.cc

```diff
@@ -31,6 +31,7 @@ using namespace pybind11::literals;
 namespace lsst {
 namespace meas {
 namespace base {
+namespace {
 
 PYBIND11_PLUGIN(algorithm) {
```

Comment:
This is different from the [examples in the style guide](https://developer.lsst.io/pybind11/style.html#wrapper-code-in-source-files-must-be-placed-in-a-nested-anonymous-namespace). Are we now required to put `PYBIND11_PLUGIN` in an anonymous namespace? I can't find any mention of anonymous namespaces in the pybind11 docs...

Reply:
I'm quite surprised. I thought our standard called for putting everything in anonymous namespace -- it's certainly easier that way, as then one can add functions as needed without worrying about it. But I have redone my changes to follow the style guide.